IM_calculation/IM/im_calculation.py

```diff
@@ -400,18 +400,12 @@ def generate_metadata(output_folder, identifier, rupture, run_type, version):
         meta_writer.writerow([identifier, rupture, run_type, date, version])
 
 
-def get_im_or_period_help(available_values, default_values, im_or_period):
+def get_im_help():
     """
-    :param available_values: All available predefined constants. Must be a superset of default_values
-    :param default_values: predefined constants
-    :param im_or_period: should be either string "im" or string "period"
-    :return: a help message for input component arg
+    :return: a help message for im arguments
     """
-    return "Available {}s are: {} and default {}s are: {}".format(
-        im_or_period,
-        ",".join(str(v) for v in available_values),
-        im_or_period,
-        ",".join(str(v) for v in default_values),
+    return "Available ims are: {} and default ims are: {}".format(
```

Comment:
The below is how it looks at the moment. We don't really need available... It's kinda said a few times.

 -m {PGA,PGV,CAV,AI,Ds575,Ds595,MMI,pSA,FAS} [{PGA,PGV,CAV,AI,Ds575,Ds595,MMI,pSA,FAS} ...], --im {PGA,PGV,CAV,AI,Ds575,Ds595,MMI,pSA,FAS} [{PGA,PGV,CAV,AI,Ds575,Ds595,MMI,pSA,FAS} ...]
                        Please specify im measure(s) separated by a space(if more than one). eg: PGV PGA CAV. Available ims are: PGA,PGV,CAV,AI,Ds575,Ds595,MMI,pSA,FAS and default ims are: PGA,PGV,CAV,AI,Ds575,Ds595,MMI,pSA